### PR TITLE
Add no-lsp flag

### PR DESCRIPTION
**Pull request by boaz-codota**

### CODE REVIEW

File: src/main/java/com/tabnine/TabNineProcess.java [hunk 1]
```diff
@@ -33,7 +32,7 @@ void startTabNine() throws IOException {
         // setup since we assume it's already built into the IDE
         List<String> command = new ArrayList<>();
         command.add(TabNineFinder.getTabNinePath());
-        List<String> metadata = new ArrayList<>();
+        List<String> metadata = new ArrayList<>(asList("--no-lsp", "true"));
```
Comment (amircodota) [inline src/main/java/com/tabnine/TabNineProcess.java:35]: This is silly, but no_lsp is not metadata. A metadata is a generic key=value parameter, that's passed as is to analytics.

It is better to add it to the command object, around lines 42

**Review by amircodota — APPROVED** [review]
Had a minor comment, but other than that LGTM

File: src/main/java/com/tabnine/TabNineProcess.java (hunk 1 above)
Comment (boaz-codota) [reply src/main/java/com/tabnine/TabNineProcess.java:35]: OK. The separation do looks silly.

**Comment by boaz-codota** [comment]
DO NOT MERGE BEFORE THE TABNINE BINARY IS IN CIRCLE FOR ALMOST EVERYONE (as this breaks the binary).
@amircodota is there even a way to control this?

**Comment by amircodota** [comment]
> DO NOT MERGE BEFORE THE TABNINE BINARY IS IN CIRCLE FOR ALMOST EVERYONE (as this breaks the binary).
> @amircodota is there even a way to control this?

@boaz-codota Not really. Experience has shown that most users upgrade pretty quickly, but there's a long tail of users that can take a long time.

What you can do is check the binary version and only add the flag after a certain version.
Not pretty, but it will work

**Comment by boaz-codota** [comment]
> > DO NOT MERGE BEFORE THE TABNINE BINARY IS IN CIRCLE FOR ALMOST EVERYONE (as this breaks the binary).
> > @amircodota is there even a way to control this?
> 
> @boaz-codota Not really. Experience has shown that most users upgrade pretty quickly, but there's a long tail of users that can take a long time.
> 
> What you can do is check the binary version and only add the flag after a certain version.
> Not pretty, but it will work

How can I do that?

**Comment by amircodota** [comment]
@boaz-codota I think intellij downloads the binary into a directory of its version something like ~/.tabnine/{version}/{platform}/TabNine. You you can check the path.

Another option - run `TabNine --version`

**Pull request merged**